bats/fb-katello-content.bats

```diff
@@ -116,6 +106,130 @@ setup() {
 }
 
+@test "export content view version" {
+  tSkipIfNoPulp3
+
+  hammer content-export complete version --organization="${ORGANIZATION}" \
+    --content-view="${CONTENT_VIEW}" --version="1.0" | grep -q "Generated"
+}
+
+@test "create skeleton org, product, content view, and repo for import" {
+  tSkipIfNoPulp3
+
+  hammer organization create --name="${IMPORT_ORG}" | grep -q "Organization created"
+  hammer product create --organization="${IMPORT_ORG}" --name="${PRODUCT}" | grep -q "Product created"
+  hammer content-view create --import-only --organization="${IMPORT_ORG}" \
+    --name="${CONTENT_VIEW}" | grep -q "Content view created"
+  hammer repository create --organization="${IMPORT_ORG}" \
+    --product="${PRODUCT}" --content-type="yum" --name "${YUM_REPOSITORY}" \
+    --url https://jlsherrill.fedorapeople.org/fake-repos/needed-errata/ | grep -q "Repository created"
+  repo_id=$(hammer --csv --no-headers repository info --name="${YUM_REPOSITORY}" --product="${PRODUCT}" --organization="${IMPORT_ORG}" --fields=id)
+  hammer content-view add-repository --organization="${IMPORT_ORG}" \
+  --name="${CONTENT_VIEW}" --repository-id=$repo_id | grep -q "The repository has been associated"
+}
+
+@test "import the exported content view" {
+  tSkipIfNoPulp3
+
+  latest_export=$(hammer --output csv --no-headers content-export list --content-view "${CONTENT_VIEW}" --organization "${ORGANIZATION}"\
+   --content-view-version="1.0" --fields="Id,Path" --per-page=1 --order="id DESC")
+  # 16,,/var/lib/pulp/exports/Test_Organization/Test_CV/1.0/2020-12-11T16-04-08-00-00,Test CV 1.0,6,2020-12-11 16:04:12 UTC,2020-12-11 16:04:12 UTC
+  export_history_id=$(echo $latest_export | cut -d, -f1) # 16
+  export_path=$(echo $latest_export | cut -d, -f2)
+  # /var/lib/pulp/exports/Test_Organization/Test_CV/1.0/2020-12-11T16-04-08-00-00
+  import_path="/var/lib/pulp/imports/bats-test-$export_history_id"
+
+  mkdir -p $import_path
+  cp -r $export_path/* $import_path
+  if [ -f "${export_path}/metadata.json" ]; then
+    metadata_path="${export_path}/metadata.json"
+  else
+    metadata_path="$(pwd)/metadata-$export_history_id.json" # metadata-16.json
+  fi
+  # no grep here because hammer doesn't output any text on success
+  hammer content-import version --content-view="${CONTENT_VIEW}" --organization="${IMPORT_ORG}"\
+    --metadata-file=$metadata_path --path=$import_path
+}
+
+@test "compare contents of export and import" {
+  tSkipIfNoPulp3
+
+  export_repos=$(hammer --output csv --no-headers content-view version show --content-view="${CONTENT_VIEW}" --organization="${ORGANIZATION}" \
+    --version="1.0" --fields="Name,Content View Name,Repositories/Name")
+  import_repos=$(hammer --output csv --no-headers content-view version show --content-view="${CONTENT_VIEW}" --organization="${IMPORT_ORG}" \
+    --version="1.0" --fields="Name,Content View Name,Repositories/Name")
```

Comment:
Recommend changing the above 3 lines to 
```bash
  export_version=$(hammer --output csv --no-headers content-view version list --content-view="${LIBRARY}" --organization="${ORGANIZATION}"\
               --per-page=1 --fields="Version")
  export_repos=$(hammer --output csv --no-headers content-view version show --content-view="${LIBRARY}" --organization="${ORGANIZATION}" \
    --version="1.0" --fields="Repositories/Name")
  import_repos=$(hammer --output csv --no-headers content-view version show --content-view="${IMPORT_LIBRARY}" --organization="${LIBRARY_IMPORT_ORG}" \
    --version="${export_version}" --fields="Repositories/Name")  
```

Reply:
@parthaa Did you mean to also put `--version="${export_version}"` for export_repos here?  you left it as `--version="1.0"`.  Just want to make sure I didn't miss something.

Reply:
That would be good idea!

Reply:
Updated to `--version="${export_version}"`